{
  "schema_version": "1.4.0",
  "id": "GHSA-r69g-j328-rwcg",
  "modified": "2024-04-04T02:47:11Z",
  "published": "2022-05-24T17:07:28Z",
  "aliases": [
    "CVE-2019-20430"
  ],
  "details": "In the Lustre file system before 2.12.3, the mdt module has an LBUG panic (via a large MDT Body eadatasize field) due to the lack of validation for specific fields of packets sent by a client.",
  "severity": [
    {
      "type": "CVSS_V3",
      "score": "CVSS:3.1/AV:N/AC:L/PR:N/UI:N/S:U/C:N/I:N/A:H"
    }
  ],
  "affected": [],
  "references": [
    {
      "type": "ADVISORY",
      "url": "https://nvd.nist.gov/vuln/detail/CVE-2019-20430"
    },
    {
      "type": "WEB",
      "url": "https://jira.whamcloud.com/browse/LU-12602"
    },
    {
      "type": "WEB",
      "url": "https://review.whamcloud.com/#/c/36208"
    },
    {
      "type": "WEB",
      "url": "http://lustre.org"
    },
    {
      "type": "WEB",
      "url": "http://wiki.lustre.org/Lustre_2.12.3_Changelog"
    }
  ],
  "database_specific": {
    "cwe_ids": [
      "CWE-20"
    ],
    "severity": "HIGH",
    "github_reviewed": false,
    "github_reviewed_at": null,
    "nvd_published_at": "2020-01-27T05:15:00Z"
  }
}